ansible/group_vars/all.yml

```diff
@@ -31,8 +31,8 @@ stanchion_data_dir: "{{ encrypted_root }}/stanchion"
 # commcare-hq connects to an S3-compatible service, which is Riak CS
 s3_blob_db_enabled: "{{ 'true' if groups.get('riakcs') else '' }}"
 s3_blob_db_url: "http://{{ groups['proxy'][0] }}:{{ riakcs_port }}"
-s3_blob_db_access_key: "{{ hostvars[riakcs_control]['riak_key'] if s3_blob_db_enabled else '' }}"
-s3_blob_db_secret_key: "{{ hostvars[riakcs_control]['riak_secret'] if s3_blob_db_enabled else '' }}"
+s3_blob_db_access_key: "{{ hostvars[riakcs_control].get('riak_key', 'not-set') if s3_blob_db_enabled else '' }}"
+s3_blob_db_secret_key: "{{ hostvars[riakcs_control].get('riak_secret', 'not-set') if s3_blob_db_enabled else '' }}"
```

Comment:
There's a lot of state floating around, so I imagine reproduction will be difficult.  I was following [these steps](https://confluence.dimagi.com/display/commcarehq/Adding+and+removing+webworkers) on new staging machines.  The command that failed was step 8 (deploy the full stack):
```bash
$ aps --limit=$NEW_MACHINES --diff --check
```